compiler/luci-interpreter/src/kernels/Conv2D.cpp

```diff
@@ -110,8 +110,15 @@ void Conv2D::configure()
     const int input_depth = input_shape.dim(3);
     Shape im2col_shape{batches, output_height, output_width,
                        input_depth * filter_height * filter_width};
-    _im2col =
-        std::make_unique<Tensor>(input()->element_type(), im2col_shape, AffineQuantization{}, "");
+    try
+    {
+      _im2col =
+          std::make_unique<Tensor>(input()->element_type(), im2col_shape, AffineQuantization{}, "");
+    }
+    catch (std::bad_alloc &ba)
+    {
+      // Failed memory allocation
```

Comment:
maybe add `_im2col = nullptr` ?

Reply:
I've added the assignment statement. Thanks.